docs/concepts/storage/persistent-volumes.md

```diff
@@ -438,12 +438,13 @@ metadata:
 provisioner: kubernetes.io/aws-ebs
 parameters:
   type: io1
-  zone: us-east-1d
+  zones: us-east-1d, us-east-1c
   iopsPerGB: "10"
 ```
 
 * `type`: `io1`, `gp2`, `sc1`, `st1`. See [AWS docs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html) for details. Default: `gp2`.
-* `zone`: AWS zone. If not specified, volumes are generally round-robin-ed across all active zones where Kubernetes cluster has a node.
+* `zone`: AWS zone. If neither zone nor zones is specified, volumes are generally round-robin-ed across all active zones where Kubernetes cluster has a node. Note: zone and zones parameters must not be used at the same time.
```

Comment:
You might want to code highlight `zone` and `zones` in these descriptions. Looks good otherwise.